<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeQQClass</class>
 <widget class="QWidget" name="WeQQClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeQQ</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>481</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="editPort">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>开始运行</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editIp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTextBrowser" name="txtLog">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCmd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WeQQ.qrc"/>
 </resources>
 <connections/>
</ui>
